### PR TITLE
fix(Core/ObjectMgr.cpp): Wrong format occurred (invalid format string)

### PR DESCRIPTION
**Pull request by cTrinity**
Wrong format occurred (invalid format string)

## Changes Proposed:
-  
-  

## Issues Addressed:

- Closes 

## Tests Performed:

- 
- 

## How to Test the Changes:

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Review by Nyeriah — APPROVED** [review]

**Review by Kitzunu — APPROVED** [review]

File: src/server/game/Globals/ObjectMgr.cpp [hunk 1]
```diff
@@ -2885,7 +2885,7 @@ void ObjectMgr::LoadItemTemplates()
             }
             if (itemTemplate.Material != dbcitem->Material)
             {
-                LOG_ERROR("sql.sql", "Item (Entry: {%u}}) does not have a correct material ({}), must be {}.", entry, itemTemplate.Material, dbcitem->Material);
```
Comment (pklloveyou) [inline src/server/game/Globals/ObjectMgr.cpp:2888]: LOG_ERROR("sql.sql", "Item (Entry: {}) does not have a correct material ({}), must be {}.", entry, itemTemplate.Material, dbcitem->Material);

File: src/server/game/Globals/ObjectMgr.cpp [hunk 2]
```diff
@@ -2897,7 +2897,7 @@ void ObjectMgr::LoadItemTemplates()
             }
             if (itemTemplate.DisplayInfoID != dbcitem->DisplayInfoID)
             {
-                LOG_ERROR("sql.sql", "Item (Entry: {%u}}) does not have a correct display id ({}), must be {}.", entry, itemTemplate.DisplayInfoID, dbcitem->DisplayInfoID);
```
Comment (pklloveyou) [inline src/server/game/Globals/ObjectMgr.cpp:2900]: LOG_ERROR("sql.sql", "Item (Entry: {}) does not have a correct display id ({}), must be {}.", entry, itemTemplate.DisplayInfoID, dbcitem->DisplayInfoID);

File: src/server/game/Globals/ObjectMgr.cpp [hunk 3]
```diff
@@ -2885,7 +2885,7 @@ void ObjectMgr::LoadItemTemplates()
             }
             if (itemTemplate.Material != dbcitem->Material)
             {
-                LOG_ERROR("sql.sql", "Item (Entry: {%u}}) does not have a correct material ({}), must be {}.", entry, itemTemplate.Material, dbcitem->Material);
+                LOG_ERROR("sql.sql", "Item (Entry: {}}) does not have a correct material ({}), must be {}.", entry, itemTemplate.Material, dbcitem->Material);
```
Comment (pklloveyou) [inline src/server/game/Globals/ObjectMgr.cpp:2888]: {}} ->  {}

File: src/server/game/Globals/ObjectMgr.cpp [hunk 4]
```diff
@@ -2897,7 +2897,7 @@ void ObjectMgr::LoadItemTemplates()
             }
             if (itemTemplate.DisplayInfoID != dbcitem->DisplayInfoID)
             {
-                LOG_ERROR("sql.sql", "Item (Entry: {%u}}) does not have a correct display id ({}), must be {}.", entry, itemTemplate.DisplayInfoID, dbcitem->DisplayInfoID);
+                LOG_ERROR("sql.sql", "Item (Entry: {}}) does not have a correct display id ({}), must be {}.", entry, itemTemplate.DisplayInfoID, dbcitem->DisplayInfoID);
```
Comment (pklloveyou) [inline src/server/game/Globals/ObjectMgr.cpp:2900]: {}} ->  {}

Comment (Nefertumm) [inline src/server/game/Globals/ObjectMgr.cpp:2900]:
```suggestion
                LOG_ERROR("sql.sql", "Item (Entry: {}) does not have a correct display id ({}), must be {}.", entry, itemTemplate.DisplayInfoID, dbcitem->DisplayInfoID);
```

**Pull request merged**